tests/test_pipeline_mixin.py

```diff
@@ -404,6 +404,11 @@ def test_pipeline_zero_shot_image_classification(self):
     def test_pipeline_zero_shot_object_detection(self):
         self.run_task_tests(task="zero-shot-object-detection")
 
+    def is_pipeline_test_to_skip(
```

Comment:
Make it a method of `PipelineTesterMixin`, so we can override easily in each model test class.

The current ugly function `is_test_to_skip` outside the class will be eventually removed once we have `is_pipeline_test_to_skip` in the necessary model test classes.